### ISSUE TITLE
Error sending NZB to Sab

### ISSUE THREAD

**Issue by mattenz** [nosmokingbandit/watcher#207]
With commit 0dd2524 I'm getting the following error when trying to grab a release:
```
INFO 2017-02-06 14:20:04,763 core.snatcher.snatch_nzb: Sending nzb to Sabnzbd.
INFO 2017-02-06 14:20:04,779 cherrypy.error.48896544.error: [06/Feb/2017:14:20:04] HTTP 
Request Headers:
  REFERER: http://server.domain:8087/status/
  ORIGIN: http://server.domain:8087
  Remote-Addr: 192.168.0.54
  Content-Length: 188
  USER-AGENT: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36 Edge/15.15025
  CONNECTION: Keep-Alive
  COOKIE: session_id=d53b255b0766c43a07ff5b96aca2e1ccbded0ea6
  X-REQUESTED-WITH: XMLHttpRequest
  HOST: server.domain:8087
  CACHE-CONTROL: no-cache
  ACCEPT: */*
  ACCEPT-LANGUAGE: en-NZ,en-US;q=0.7,en;q=0.3
  Content-Type: application/x-www-form-urlencoded; charset=UTF-8
  ACCEPT-ENCODING: gzip, deflate, br
ERROR 2017-02-06 14:20:04,779 cherrypy.error.48896544.error: [06/Feb/2017:14:20:04] HTTP 
Traceback (most recent call last):
  File "c:\watcher\lib\cherrypy\_cprequest.py", line 670, in respond
    response.body = self.handler()
  File "c:\watcher\lib\cherrypy\lib\encoding.py", line 220, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "c:\watcher\lib\cherrypy\_cpdispatch.py", line 60, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "c:\watcher\core\ajax.py", line 294, in manual_download
    return json.dumps(self.snatcher.snatch(data))
  File "c:\watcher\core\snatcher.py", line 91, in snatch
    response = self.snatch_nzb(data)
  File "c:\watcher\core\snatcher.py", line 120, in snatch_nzb
    response = sabnzbd.Sabnzbd.add_nzb(data)
  File "c:\watcher\core\downloaders\sabnzbd.py", line 52, in add_nzb
    conf = core.CONFIG['Downloader']['Torrent']['Sabnzbd']
KeyError: 'Sabnzbd'
```
It's getting a list of NZB's ok however when I pick one to send to SAB (v1.2) the GUI appears to be doing nothing but if I look at the logs I see the above.

**Comment by mjdgoldeneye** [comment]
I can say I'm getting the same error. Not sure why it has ['Torrent'] there, as we're working with nzbs.

**Comment by nosmokingbandit** [comment]
Global find/replace is always a bad idea...

Fixed in 78eada88dd4d9c2926cda4d87da914cbb93becc7
